### DISCUSSION TITLE
Separate Updates to Dependencies from Subteams

### DISCUSSION THREAD

**Discussion by anopheles** [renovatebot/renovate#14934] [Request Help] [answered]
Hi everyone :wave:,

I'm currently setting up renovate for a repository that is used by multiple subteams. The directory structure looks something like this

```
./renovate.json
./sub-team-1/
./sub-team-2/
./sub-team-3/
...
```

my renovate config currently looks similar to this
```json
{
  "extends": ["config:base", ":disableDependencyDashboard"],
  "groupName": "all",
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "enabledManagers": ["npm"],
  "baseBranches": ["master"],
  "includePaths": [
    "sub-team-1/**",
    "sub-team-2/**"
  ],
  "packageRules": [
    {
      "paths": [
        "sub-team-1/**"
      ],
      "assignees": ["anopheles"],
      "labels": ["sub-team-1"]
    },
    {
      "paths": [
        "sub-team-2/**"
      ],
      "assignees": ["anopheles"],
      "labels": ["sub-team-2"]
    }
  ],
  "npm": {
    "extends": ["helpers:disableTypesNodeMajor", "config:js-app"],
    "postUpdateOptions": ["npmDedupe"]
  },
  "labels": ["renovate"]
}
```

Unfortunately, renovate creates PRs that update dependencies in both folders, a specific example is a PR called "Pin dependencies" that pines "@types/jest". 

Is there an easy way to completely separate the PRs on a subfolder? Is the problem perhaps because I use "config:base" which adds the [monorepo presets](https://docs.renovatebot.com/presets-monorepo/)?.

Cheers,
anopheles

**Comment by astellingwerf** [comment, marked as answer]
Would it help to add the [`additionalBranchPrefix` property](https://docs.renovatebot.com/configuration-options/#additionalbranchprefix) to the package rules? You may want to use its example, or use `"sub-team-1-"` explicitly to keep it possible to group the same dependencies in different files within a teams folder.

**Reply by anopheles** [reply]
Thanks! That's exactly what I was looking for!
